README.md

```diff
@@ -49,6 +54,18 @@ user , error = xee.get_user(token.access_token)
 print(user.id)
 ```
 
+Others examples:
+
+```python
+status , error = xee.get_status(carId,token.access_token)
+print(status)
+```
+
+```python
+signal , error = xee.get_signals(carId,token.access_token,names=['odometer', 'FuelLevel'])
```

Comment:
Oups seems to be typo error. Let me correct this before merging.

Le 14 septembre 2016 18:56:33 GMT+02:00, Quentin Klein <notifications@github.com> a écrit :
>> @@ -49,6 +54,18 @@ user , error = xee.get_user(token.access_token)
>>  print(user.id)
>>  ```
>>  
>> +Others examples:
>> +
>> +```python
>> +status , error = xee.get_status(carId,token.access_token)
>> +print(status)
>> +```
>> +
>> +```python
>> +signal , error =
>xee.get_signals(carId,token.access_token,names=['odometer',
>'FuelLevel'])
>
>Real signal is `Odometer`, but I think it works fine like this.
>
>-- 
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/quentin7b/xee-sdk-python/pull/18/files/aca3962ebb00538ac1534771e2749e150a136895#r78789337

-- 
Envoyé de mon appareil Android avec K-9 Mail. Veuillez excuser ma brièveté.